### PR TITLE
Fix various 1.18.2 update issues

### PR DESCRIPTION
**Pull request by Machine-Maker**

### CODE REVIEW

File: patches/server/0638-More-World-API.patch [hunk 1]
```diff
@@ -3,12 +3,72 @@ From: Jake Potrebic <jake.m.potrebic@gmail.com>
...
+             //this.getPlayerList().saveAll(); // Paper - we don't need to do this
+diff --git a/src/main/java/net/minecraft/server/level/ServerLevel.java b/src/main/java/net/minecraft/server/level/ServerLevel.java
+index 1b678f60ae054f6d9c97ba17f5ffe258d65cccb1..72e931564e0b9d3f51dc8dcfff443a079e554001 100644
+--- a/src/main/java/net/minecraft/server/level/ServerLevel.java
++++ b/src/main/java/net/minecraft/server/level/ServerLevel.java
+@@ -1631,7 +1631,16 @@ public class ServerLevel extends Level implements WorldGenLevel {
+             if (optional.isEmpty()) {
+                 return null;
+             } else {
+-                Pair<BlockPos, Holder<ConfiguredStructureFeature<?, ?>>> pair = this.getChunkSource().getGenerator().findNearestMapFeature(this, (HolderSet) optional.get(), pos, radius, skipExistingChunks);
```
Comment (electronicboy) [inline patches/server/0638-More-World-API.patch]: ?

Comment (electronicboy) [reply patches/server/0638-More-World-API.patch]: nvm, i see

**Review by electronicboy — APPROVED** [review]

File: patches/server/0638-More-World-API.patch [hunk 2]
```diff
@@ -3,12 +3,72 @@ From: Jake Potrebic <jake.m.potrebic@gmail.com>
 Date: Tue, 7 Jul 2020 10:52:34 -0700
 Subject: [PATCH] More World API
 
+1.18.2: Fixes an issue with World#locateNearestStructure
```
Comment (jpenilla) [inline patches/server/0638-More-World-API.patch]: I see how it kinda fits, but this could probably just go in it's own patch
